src/v2/Apps/Fair/Routes/FairArtworks.tsx

```diff
@@ -46,8 +44,7 @@ const FairArtworksFilter: React.FC<FairArtworksFilterProps> = props => {
 
   const Filters = () => (
     <Box pr={2}>
-      {!!user && <FollowedArtistsFilter />}
-
+      <ArtistsFilter />
```

Comment:
Renamed it to `ArtistsFilter` since this now includes the 'special' followed artist filter _and_ any artist aggregation filters.